tools/Tool.cc

```diff
@@ -51,6 +57,61 @@ static bool outputIsColor() {
     return sColor;
 }
 
+static bool tokenize(const char* input, deque<string> &args)
+{
+    string str(input);
+    int start = 0;
+    for( size_t i=0; i<str.length(); i++){
+        char c = str[i];
+        if( c == ' ' ) {
+            if(i - start > 1) {
```

Comment:
This test seems wrong. If the first argument is 1 byte long, like `x y`, then this test will be false and the first argument will be ignored. Initializing `start` to -1 might fix this.